source/common/upstream/load_balancer_impl.h

```diff
@@ -66,6 +83,17 @@ class LoadBalancerBase {
   std::vector<uint32_t> per_priority_health_;
 };
 
+class LoadBalancerContextBase : public LoadBalancerContext {
```

Comment:
Can we potentially add defaults of all the methods? I think there is other boilerplate you could likely delete from all the call sites?